pyartcd/pyartcd/pipelines/update_golang.py

```diff
@@ -40,15 +40,9 @@ def is_latest_build(ocp_version: str, el_v: int, nvr: str, koji_session) -> bool
     if nvr == latest_build[0]['nvr']:
         _LOGGER.info(f'{nvr} is the latest build in {build_tag}')
         return True
-
-    if el_v == 8 and 'module' in nvr:
-        _LOGGER.info(f'{nvr} is not the latest build in build tag {build_tag}. Since it is a module build, '
-                     f'we need to update tag inheritance. Use the flag --create-tagging-ticket to create a jira '
-                     'ticket to do that.')
-    else:
-        override_tag = f'rhaos-{ocp_version}-rhel-{el_v}-override'
-        _LOGGER.info(f'{nvr} is not the latest build in {build_tag}. Run `brew tag {override_tag} {nvr}` to tag the '
-                     f'build and then run `brew regen-repo {build_tag}` to make it available.')
+    override_tag = f'rhaos-{ocp_version}-rhel-{el_v}-override'
+    _LOGGER.info(f'{nvr} is not the latest build in {build_tag}. Run `brew tag {override_tag} {nvr}` to tag the '
+                 f'build and then run `brew regen-repo {build_tag}` to make it available.')
```

Comment:
I suggest we remove this message from here and instead move it to main and tweak it to "run with --tag-builds"

Reply:
I think the tag operation will usually be a successful step, and if run with the tag build parameter, it probably won't come here